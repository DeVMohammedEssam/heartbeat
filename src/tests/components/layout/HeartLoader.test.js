import { shallow } from 'enzyme';
import React from 'react';
import HeartLoader from '../../../components/layout/HeartLoader';
test('should render hearbearloadr component correctly', () => {
	const wrapper = shallow(<HeartLoader />);
	expect(wrapper).toMatchSnapshot();
});
