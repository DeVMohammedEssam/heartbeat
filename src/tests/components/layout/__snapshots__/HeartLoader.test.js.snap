// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render hearbearloadr component correctly 1`] = `
<div
  className="heart-loader"
>
  <div
    className="heart-loader__animation-container"
  >
    <span
      className="heart-loader__dashes"
    />
    <span
      className="rotator"
    />
    <div
      className="heart-loader__img-container"
    >
      <img
        alt=""
        src="/images/heart-icon.png"
      />
    </div>
  </div>
</div>
`;
